### DISCUSSION TITLE
Problems changing the Title of Collections

### DISCUSSION THREAD

**Discussion by ouhumanitieslibrarian** [knowledgefutures/pubpub#1413] [Support] [answered]
Greetings all,

Firstly, I want to say how grateful I am for all the work you folks are doing on the PubPub platform -  I really appreciate the commitment to open scholarly publishing and its messy particularities.

I'm working with the Performance of the Real community on their journal and we're having problems changing the title of one of our collections - as an example [Performing Ecologies](https://performancereal.pubpub.org/performing-ecologies-2021).

I managed to change it once, but found it hard to work out change it again. With a little bit of experimentation, it looks like I can just change the title of the collection or other details and it updates / auto-saves; however if I want to change the DOI or ISSN or other metadata, I need to save my changes. Is that right?

What I find confusing is that if I change the title and then change something in the other metadata AND try and save those changes, it reverts the title to the previous version, even though my attempted change is still showing in the title text entry box ... anyone else found this?

Many thanks,
Alexander

**Comment by gabestein** [comment, marked as answer]
Hi Alexander,

Thanks for posting, and I'm sorry you're running into the issue. This is bad UX on our part that we know we need to fix, but don't know exactly when yet. You can see some discussion of it [here](https://github.com/pubpub/pubpub/issues/1255), where I submitted the same issue (pre-forum).

Long story short, the workaround is to refresh the page after changing the title.

Shorter story long: what's happening is you change the title and it auto-saves. But that auto-save doesn't make a round trip back to the browser, so when you update the metadata, it's using the same old data again. Hence, update the title and then refresh.

**Reply by ouhumanitieslibrarian** [reply]
Thanks Gabe.

Yes that makes sense - after some playing with the page, I wondered if that was the case.
Only a minor annoyance now that I understand what the page is trying to do :)

**Reply by writingpoli** [reply]
Danke! It was definitely unexpected.
